app/src/main/java/com/pramod/dailyword/framework/datasource/network/mappers/WordNEMapper.kt

```
@@ -34,11 +37,63 @@ class WordNEMapper @Inject constructor() : EntityMapper<WordNE, Word> {
+            antonyms = entity.antonyms,
+            bookmarkedId = null,
+            bookmarkedAt = null,
+            bookmarkedSeenAt = null,
+            isSeen = false,
+            seenAtTimeInMillis = null,
+            wordHistory = wordHistory
+        )
+    }
+
+    override suspend fun toEntity(domain: Word): WordNE {
+        TODO("Not yet implemented")
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)